promtail/README.md

```diff
@@ -0,0 +1,29 @@
+# Promtail
+
+## Usage
+
+Run using docker:
+
+```sh
+docker run -v /var/run/docker.sock:/var/run/docker.sock \
+    -e LOKI_PUSH_URL="<domain>/loki/api/v1/push" \
+    -e LOKI_PUSH_USER="<user>" \
+    -e LOKI_PUSH_PASSWORD="<password>" \
+    -e DOCKER_HOST="/var/run/docker.sock" \
+    -e INSTANCE_NAME="HelloWorld" \
+    scalableminds/promtail
+```
+
+This will scrape all containers with label `gather.logs=true`.
+
+## Configuration
+
+Environment Variables:
+
+| Name | Description |
+|------|-------------|
+| `LOKI_PUSH_URL` | URL to push to (e.g. `<domain>/loki/api/v1/push`) |
+| `LOKI_PUSH_USER` | User for Basic Auth |
```

Comment:
In metrics-pusher, this is called AUTH_USER and AUTH_PASSWORD. Let's make it consistent